### PR TITLE
views/project count on user - has MIGRATION; resolves #1681

### LINKED ISSUE #1681: Bring stats in user model to browser

### PR DESCRIPTION
**Pull request by kschzt**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Vizor",
-  "version": "0.10.3",
+  "version": "0.11.0",
```
Comment (gmarinov) [inline package.json]: do we need this? engine stays the same..

**Comment by gmarinov** [comment]
see version comment, otherwise lgtm

**Comment by kschzt** [comment]
@gmarinov I'm upping the minor version, because the software stops working properly without the migration script.

**Comment by gmarinov** [comment]
lgtm

**Pull request merged**